azplugins/test-py/test_pair_hertz.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2018-2020, Michael P. Howard
+# This file is part of the azplugins project, released under the Modified BSD License.
+
+# Maintainer: arjunsg2
+
+from hoomd import *
```

Comment:
This may be copied from elsewhere, but please change this to:
```py
import hoomd
```
so that you aren't importing everything from the hoomd namespace. You may need to modify some of the things below (`data.`, `init.`, `context.`).

Reply:
I tried to do it this way, but after I did this and changed the functions below to include the extra 'hoomd.' the unit test would not pass. Unfortunately I could not isolate the source of this discrepancy (even after trying to add the 'hoomd.' to other things you didn't mention such as hoomd.run(), hoomd.group(), etc.). As you stated I copied this from elsewhere and was using this as a template.

Reply:
OK, no problem. You probably just missed one or two statements, but this will be fixed as part of a larger refactoring of the pair potential unit tests later.

Reply:
I've made some changes to the unit test, please have a look. The unit test was failing because after changing the import statement, the following code has to reference the classes explicitly, e.g.  ``` context. ``` --> ```hoomd.context``` etc. The easiest way to find those things is to run only one unit test with verbose output: ``` ctest -V -R azplugins-test_pair_hertz-cpu``` and then look from the top for error messages like: ```NameError: name 'context' is not defined``` and fix them one by one.

Reply:
Thank you Antonia!